.github/workflows/job_PreCommit_PythonAutoformatter.yml

```diff
@@ -0,0 +1,27 @@
+name: Precommit Python Formatter
+run-name: Python Formatter
+on:
+  pull_request:
+    branches: [ "master", "release-*" ]
+    paths: [ "sdks/python/apache_beam/.*"]
+  issue_comment:
+    types: [created, edited]
+
+
+jobs:
+  PythonFormatter:
+    if: github.event.issue.pull_request && contains(github.event.comment.body, 'Run PythonLint PreCommit')
+    runs-on: self-hosted
+
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.7 - 3.9'
+      - name: Install Dependencies
+        run:  pip install --retries 10 --upgrade pip && ls && pwd && pip install --retries 10 --upgrade tox==3.20.1 -r ./build-requirements.txt
```

Comment:
We might need a single source-of-truth for tox version instead of every yaml having this line. Or call  existing gradle task `:pythonFormatterPreCommit` (see below)